src/Policeman/Evaluate.hs

```diff
@@ -0,0 +1,30 @@
+{- | This is the evaluation step.
+
+We now need to calculate the next version based on the given 'PackageDiff' and
+modify the previous version accordingly.
+-}
+module Policeman.Evaluate
+    ( eval
+    ) where
+
+import Policeman.Core.Diff (PackageDiff (..), emptyDiff, hasDiffAdded, hasDiffDeleted)
+import Policeman.Core.Package (Version, bumpMajorVersion, bumpMinorVersion)
+
+
+{- | Generates the next version in agreement with PVP.
+-}
+eval :: Version -> PackageDiff -> Version
+eval ver PackageDiff{..}
+    | moduleDiffNonEmpty = bumpMajorVersion ver
+    | removedExport = bumpMajorVersion ver
+    | newExport = bumpMinorVersion ver
+    | otherwise = ver
```

Comment:
Nice basic implementation of this function! And it's nicely written with guards :slightly_smiling_face: 
I guess this implementation is ok for now, and more detailed report on why this version should be bumped will be added later.